cfme/tests/services/test_provision_stack.py

```diff
@@ -38,8 +38,10 @@ def template(provider, provisioning, setup_provider):
 
     if provider.type == "ec2":
         data_file = load_data_file(str(orchestration_path.join('aws_vm_template.json')))
-    elif provider.type == "openstack":
+    elif provider.name == "RHOS7-GA":
```

Comment:
**REQUIRED:** There should never be private data in a test file. If there is a required data file it can be referenced in the yaml, but we shuldn't ever put keys in the test. This means the test would only work if these keys were present. Which may not always be true and certainly won't be true if dev start running our tests.